### ISSUE TITLE
 Cannot read properties of undefined (reading 'safelistRequestStatus')

### ISSUE THREAD

**Issue by JohnPochta** [alchemyplatform/alchemy-sdk-js#416]
### [REQUIRED] Environment

- Alchemy SDK version: 3.2.0

### [REQUIRED] Describe the problem
I have the code snippet:

```
this.alchemy = new Alchemy({
    url: this.NETWORK_RPC_URL,
  });

return await this.alchemy.nft.getNftsForOwner(address, {
  omitMetadata: false,
});
```

This throws the error:  Cannot read properties of undefined (reading 'safelistRequestStatus') .
I figured out that it's related to OpenSea metadata parsing and most likely someone forgot smth like: 
`
rawNftContract.openSeaMetadata**?**.safelistRequestStatus 
`
in src/utils/util.ts.

#### How to reproduce:

```
this.alchemy = new Alchemy({
      url: this.NETWORK_RPC_URL,
    });

return await this.alchemy.nft.getNftsForOwner(address, {
  omitMetadata: false,
});
```

**Comment by nirmit1509** [comment]
@JohnPochta Can you please escalate this issue. There is an open PR for the same as well. Thanks in advance 🙏

**Comment by thebrianchen** [comment]
Sorry for the delay. This fix has been applied and published in `v3.3.1`.

**Comment by nirmit1509** [comment]
https://github.com/alchemyplatform/alchemy-sdk-js/pull/432

**Comment by nirmit1509** [comment]
@thebrianchen The comparison with null was also wrong.
Can you please fix this as well.
https://github.com/alchemyplatform/alchemy-sdk-js/pull/432

**Comment by anthhub** [comment]
@thebrianchen The comparison with null was also wrong.
Can you please fix this as well.
https://github.com/alchemyplatform/alchemy-sdk-js/pull/432

**Comment by thebrianchen** [comment]
@anthhub @nirmit1509 Sorry for the delay in not fixing this. This is now fixed in `v3.4.1`.
